File: BackEnd/BackEnd/Domain/Models/Cuestionario.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace BackEnd.Domain.Models
{
    public class Cuestionario
    {
        public int Id { get; set; }

        [Required]
        [Column(TypeName ="varchar(100)")]
        public string Nombre { get; set; }

        [Required]
        [Column(TypeName = "varchar(150)")]

        public string Descripcion { get; set; }

        public DateTime FechaCreacion { get; set; }

        public int Activo { get; set; }

        public int UsuarioId { get; set; }

        public Usuario Usuario { get; set; }

        public List<Pregunta> listPreguntas { get; set; }

    }
}

File: FrontEnd/src/app/models/cuestionario.ts
import { Pregunta } from "./pregunta";

export class Cuestionario{
    id?: number;
    nombre: string;
    Descripcion: string;
    fechaCreacion?: Date;
    listPreguntas: Pregunta[];

    constructor(nombre: string, Descripcion: string, fechaCreacion: Date, listPreguntas: Pregunta[]){

        this.nombre=nombre;
        this.Descripcion=Descripcion;
        this.fechaCreacion=fechaCreacion;
        this.listPreguntas=listPreguntas;
    }

}
